### DISCUSSION TITLE
Severity Dropdown not showing on Chrome

### DISCUSSION THREAD

**Discussion by steve9825679** [opcodesio/log-viewer#333] [Q&A] [answered]
Hello!

On Chrome, the severity dropdown is not showing, the search input expands to the whole width. In Edge, Firefox and Opera the dropdown is shown correctly.

When I remove the "none" class on the button (id "severity-dropdown-toggle"), the dropdown shows and works (although it is styled without border and bg-color). 
And this cannot be the problem, because on eg. Firefox the none-class is set too although the dropdown is visible.

Any idea, what I am doing wrong here? I am on version 3.1.11 and published the assets. Thanks for any help! Stefan

![2024-02-14 20_29_27-Log Viewer - SINTOORA](https://github.com/opcodesio/log-viewer/assets/45850599/fb03d38a-4239-4909-a745-c716de19b254)

**Comment by arukompas** [comment]
hey @steve9825679 

First of all, have you tried clearing the browser cache? If it works in other browsers, but not Chrome, then maybe something's cached incorrectly.

Second, you can try upgrading to the latest version (don't commit to it, just see if it helps locally).

And third, if none of the above help, can you share what CSS styles are being applied to the element `#severity-dropdown-toggle` ? See below screenshot for example. The `none` class should only affect the colour of the badge, not its visibility.

<img width="1435" alt="Screenshot 2024-02-15 at 18 24 35" src="https://github.com/opcodesio/log-viewer/assets/8697942/34e74a9c-7f1e-49bf-a90f-73f7b0e1a659">

**Comment by steve9825679** [comment]
Hi, thanks a lot for your suggestions. Cache is cleared, and turned off in the dev-tools, no change. I updated to v3.4, no change. Here is a screenshot from the applied classes... at the very bottom is one called ".none" with "display:none !important", which I can't even remove in the dev-tools. But this one seems to overwrite the display:inline-flex in the .badge-class...

edit: in other browsers, this "injected stylesheet" is not present... perhaps this helps anyway?

![opcodesio02](https://github.com/opcodesio/log-viewer/assets/45850599/c3eedf09-c251-4c13-b912-2313b95ae593)

**Reply by arukompas** [reply, marked as answer]
@steve9825679 try disabling all addons/extensions in Chrome. Injected stylesheets can come from extensions trying to modify page's look or behaviour, or maybe it's just a style used on one of the extensions.

If you can figure out which extension it is, we can report this as a bug to the extension developer - their code should not affect existing pages, their styles should be scoped.

**Comment by steve9825679** [comment]
You're right! It was the ChatGPT Extension for Chrome! Uinstalled, and log-viewer is working perfectly fine!
Thanks a lot, never would have guessed that!
And: I loooove the log-viewer, have it installed on every application ;-)

**Reply by arukompas** [reply]
Happy to hear! 🤓🥳🎉
